City-of-Pori/pori-kada
# Pori 88 shortcuts block

To test:

1. drush fra -y && drush cc all
2. Go to edit front page node and add shortcuts to it (Oikopolut)
3. Save and check the front page that it has shortcuts block visible over the slideshow and it works on mobile and desktop
4. Do the same for visitpori front page.

Designs:
Desktop for Visit Pori: https://drive.google.com/file/d/0B7ZHsug7PcEDc1oyQXkxdjdoNWc/view
Mobile for Visit Pori: https://drive.google.com/file/d/0B7ZHsug7PcEDWDdKN1hvaE9jNW8/view
Design for Pori.fi is the same as Visit Pori but with the color black.
